plugins/explore/README.md

```diff
@@ -79,3 +79,37 @@ const routes = (
   </FlatRoutes>
 );
 ```
+
+## ToolExplorer Content Customization
+
+Override the `exploreToolsConfigRef` API in `/packages/app/src/apis.ts`.
+
+```tsx
+import { exploreToolsConfigRef } from '@backstage/plugin-explore-react';
+
+export const apis: AnyApiFactory[] = [
+  ...
+
+  createApiFactory({
+    api: exploreToolsConfigRef,
+    deps: {},
+    factory: () => ({
+      /*  pass the tools array
+          i.e. tools = [
```

Comment:
Please double check to make sure this works x)

Reply:
@Rugvip it works fine. implemented in our application :)

Reply:
What does the code look like in your app?

Reply:
is this what u meant?
```
factory: () => ({
      async getTools() {
        return tools;
      },
    }),

```where tools is the content array,

Reply:
Yes! let's have something like that instead. Important part here being the async `getTools` function.